invisible_cities/reco/psf_functions_test.py

```diff
@@ -55,6 +55,7 @@ def test_add_empty_sensors_and_normalize_q_file(ICDATADIR):
                                  database = load_db.DataSiPM('new', 0))
     hdst_processed.reset_index(inplace=True, drop=True)
     hdst_psf       = pd.read_hdf(PATH_TEST)
+    hdst_psf       = hdst_psf.astype(hdst_processed.dtypes) # since pandas version update
```

Comment:
Minor point, but I'd mention the old and new version in the comment about the pandas version update.

And again when the same comment appears on line 72